### PR TITLE
Docs: reflow the text and add better markup

### PR DESCRIPTION
**Pull request by jsarenik**
Here are some very uninportant fixes that just make the modified files look a bit more professional (one in rendered output, the other in terminal as plain-text).

No changes to the text were done here. Just markup (monospace, multi-line links) and `fmt`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,26 +29,33 @@ To contribute a patch, the workflow is as follows:
...
 
 Commit messages should be verbose by default consisting of a short subject line
 (50 chars max), a blank line and detailed explanatory text as separate
 paragraph(s), unless the title alone is self-explanatory (like "Corrected typo
 in init.cpp") in which case a single title line is sufficient. Commit messages should be
 helpful to people reading your code in the future, so explain the reasoning for
-your decisions. Further explanation [here](http://chris.beams.io/posts/git-commit/).
+your decisions. Further explanation [here].
+
+[here]: http://chris.beams.io/posts/git-commit/
```
Comment (fanquake) [inline CONTRIBUTING.md]: If we're touching this please make this link https

Comment (jsarenik) [reply CONTRIBUTING.md]: Thanks for suggestion. As the https link works, there it is now.

**Comment by fanquake** [comment]
Going to close this for now. Reflowing and reformatting text isn't a priority.

**Comment by jsarenik** [comment]
@fanquake Fair enough, I see your point. In order to hit the priorities I think that one needs textual documentation which will be as readable as possible.

See [original](http://h.jasan.tk:81/original.png) and [reflowed](http://h.jasan.tk:81/reflowed.png) screenshots.

**Pull request closed without merging**